sdk/monitor/Azure.Monitor.Ingestion/src/Models/UploadLogsOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Serialization;
+
+namespace Azure.Monitor.Ingestion
+{
+    /// <summary>
+    /// The options model to configure the request to upload logs to Azure Monitor.
+    /// </summary>
+    public class UploadLogsOptions
+    {
+        /// <summary>
+        /// The serializer to use to convert the log objects to JSON.
```

Comment:
Should we mention what the default serializer will be? I think it would be JsonObjectSerializer which uses System.Text.Json.